hw_5/hw_5.2.js

```diff
@@ -0,0 +1,130 @@
+/* 
+Task 2.
+1. Написать скрипт, переводящий количество байт в нужные единицы
+  bytes => kB Mb Gb Tb
+  16 565 846 bytes (16,6 Mb)
+
+  1 Kb = 1024 byte
+  1 Mb = 1024 Kb
+  1 Gb = 1024 Mb
+  1 Tb = 1024 Gb
+
+  // Пример: ~ 1000
+  4 548 = 4,5 Kb (Real 4,4 Kb)
+  454 548 = 454,5 Kb
+  1 454 548 = 1,5 Mb
+
+  Результат должен быть округлен до 1 знака после запятой методом .toFixed(), про который надо почитать самим ;)
+
+2. Сделать из "*" в консоли равнобедренный треугольник и ромб
+
+3.  Вам нужно вывести в консоль числа от 1 до 100.
+    Если число делится без остатка на 3, то выведете в консоль “число - делится на 3”.
+    Если число делится на 5 без остатка, то то выведете в консоль “число - делится на 5”.
+    Если число делится и на 3 и на 5 без остатка, то то выведете в консоль “число - делится и на 3 на 5”.
+    Число 15 делится без остатка на 3 и на 5 -- пример сообщения в консоле.
+
+4. Написать скрипт, который преобразует любое предложение в camelCase. Первое слово должно начинаться с буквы в нижнем регистре, 
+  у остальных -  верхнем. Пример: I am super engineer => iAmSuperEngineer
+*/
+
+// subtask 1
+const bytes =  1456235400000;
+const divider = 1024;
+let counter = 0;
+
+while (true) {
+	if (bytes / divider >= 1) {
+		bytes /= divider;
+		counter++;
+	} else break;
+}
+
+switch (counter) {
+	case 0: {
+		console.log(`You don't need converting. The result is ${bytes} bytes.`);
+		break;
+	}
+	case 1: {
+		console.log(`The result is ${bytes.toFixed(1)} Kb.`);
+		break;
+	}
+	case 2: {
+		console.log(`The result is ${bytes.toFixed(1)} Mb.`);
+		break;
+	}
+	case 3: {
+		console.log(`The result is ${bytes.toFixed(1)} Gb.`);
+		break;
+	}
+	case 4: {
```

Comment:
In case of greater initial number, counter value could be 5 or even more. You need to check if it is 4 or more, so gigabites

Reply:
changes done